templates/home.html

```diff
@@ -1,16 +0,0 @@
-{% extends 'layout.html' %}
-
-{% block app_content %}
-{% if current_user.is_active %}
-<h1 class="text-center">Welcome <bf class="text-capitalize">{{ current_user.first_name }}</bf>!</h1>
-<hr>
-<p class="text-center">You are logged in and ready to ride in <bf class="text-capitalize">{{ city }}</bf> (GCP region <code>{{ region }}</code>).</p>
```

Comment:
I've found having location information displayed somewhere to be really helpful. Since https://github.com/cockroachlabs/movr-flask/pull/8, clients are assigned to specific regional deployments that correspond to the GCP region displayed here. Thoughts on adding it to the navbar?

Reply:
This is worth bringing up with @jeremycbloom 

To summarize:

The visual update removes the home page nav, but @ericharmeling brings up that having location information displayed somewhere is helpful, and that there's logic in the background to determine one's default region.

Thinking aloud here:
- we could bring the Home page back (so visual update could include a "Home" nav)
- we could remove the Home page/nav, but somehow incorporate that location information on other pages (perhaps the vehicles page, since that's when you're looking at vehicles in your area).

Thoughts @jeremycbloom ?

cc @elkmaster

Reply:
I'll mock something up. Thanks for the context!

Reply:
@celiala @ericharmeling 
![image](https://user-images.githubusercontent.com/24399608/87688991-ac97d280-c755-11ea-98d9-3e2279075b40.png)

Reply:
I've got no attachment to the homepage other than that location information (city + region), which Jeremy captures really well in this navbar mockup.

Reply:
cc @thatnerd